tools_docker/Debian_Uwsgi/entrypoint.sh

```diff
@@ -106,27 +156,29 @@ sed -i "s|^;\s*app.enabled = .*|app.enabled = $DEFAULT_APP_LIST|g" /etc/tracim/d
 # TODO PA 2019-06-19 Rework the index-create part according to https://github.com/tracim/tracim/issues/1961
 # Make sure index is created in case of Elastic Search based search. (the command does nothing in case of simple search)
 cd /tracim/backend/
-tracimcli search index-create -c /etc/tracim/development.ini
-
+tracimcli_as_user "search index-create"
 
+log "Start all services"
 # starting services
 service pushpin start # tracim live messages (TLMs) sending
 service zurl start # tracim live messages (TLMs) sending
 service redis-server start  # async jobs (for mails and TLMs)
 service apache2 restart
-
-supervisord -c /tracim/tools_docker/Debian_Uwsgi/supervisord_tracim.conf
+log "Run supervisord"
+supervisord -c "$docker_script_dir/supervisord_tracim.conf"
 # Activate daemon for reply by email
 if [ "$REPLY_BY_EMAIL" = "1" ];then
+    log "start mail fetcher"
     supervisorctl start tracim_mail_fetcher
 fi
-
 # Start tracim
+log "start uwsgi"
 set +e
 service uwsgi restart
 set -e
 if [ "$START_CALDAV" = "1" ]; then
+    log "start caldav"
```

Comment:
Maybe something like "Synchronize caldav file/account". Caldav is not started here.

Reply:
OK, I'll update the comment.